### ISSUE TITLE
DockerListImages Filters

### ISSUE THREAD

**Issue by DavyC** [bmuschko/gradle-docker-plugin#412]
The change in https://github.com/bmuschko/gradle-docker-plugin/commit/b1f15e68f38abc902fd402956b639ddb78b76e41#diff-3a6946d2064fc3e5dedea41fecfc477a modified the filters to label-only filters. That seems to be a mistake?

**Comment by cdancy** [comment]
@DavyC did something pop for you that you're mentioning this?

@orzeh any thoughts?

**Comment by orzeh** [comment]
@DavyC this change was made because `docker-java` API changed in `3.0.0` (see [this commit](https://github.com/docker-java/docker-java/commit/acfe65de6f4ef1392a9bc561a01d88268ef9869b)). Indeed we now support only label filtering and other filters are missing. Are you interested in submitting PR?

**Comment by DavyC** [comment]
Yeah, I was playing around this plugin and followed the example I found in the functional test: https://github.com/bmuschko/gradle-docker-plugin/blob/master/src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerListImagesFunctionalTest.groovy#L28

It took me awhile to figure out what happened. I thought I did something wrong in the usage/syntax. Eventually, I'd like to use the `reference` filter option.

I will submit a PR and let me know what you guys think.
